CONTRIBUTING.md

```diff
@@ -37,6 +37,8 @@ Below is a short excerpt of some points in there:
 
     The old config generation system used impure shell scripts and could break in specific circumstances (see #1234).
 
+  If you add yourself as maintainer, use `maintainers: add <handle>` as commit message ([see also](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers) and as a seperate commit.
```

Comment:
```suggestion
  In case of adding yourself as maintainer, do it in a separated commit (no need for a Pull Request), with message `maintainers: add <handle>`. See also ([the section of manual about meta.maintainers](https://nixos.org/manual/nixpkgs/stable/#var-meta-maintainers).
```

Reply:
It isn't even necessarily adding yourself. It's adding anyone to the maintainer list.

Reply:
This is the most common scenario: a newcomer adding a package.
But point taken, I will update this.